### PR TITLE
Localise the footer links to point to the Localised WordPress site when it exists.

### PR DESCRIPTION
**Pull request by dd32**
See #52

I've gone with a filter rather than defining a custom array of nav links for ease of future updating; both of the links on the page and more complex logic for the destination url.

### CODE REVIEW

**Comment by dd32** [comment]
A diff of the change, ignore the classname & script changes that happens with block output. I've removed some `<script>`  chunks and purely-classname chunks from the diff.

command: `diff -U1 <(curl -s https://wordpress.org/wp-json/global-header-footer/v1/footer) <(curl -s https://de.wordpress.org/wp-json/global-header-footer/v1/footer) | colordiff`


```diff
--- /dev/fd/11	2022-01-20 15:53:52.000000000 +1000
+++ /dev/fd/13	2022-01-20 15:53:52.000000000 +1000
@@ -2,14 +2,14 @@

-<footer class="wp-container-61e8f8f27a3b9 wp-block-group global-footer alignfull">
+<footer class="wp-container-61e8f8f486bd0 wp-block-group global-footer alignfull">

-	<div class="wp-container-61e8f8f2716f0 wp-block-group global-footer__navigation-container">
-		<nav class="wp-container-61e8f8f25df04 global-footer__navigation-important wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/about/"><span class="wp-block-navigation-item__label">About</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/news/"><span class="wp-block-navigation-item__label">Blog</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/hosting/"><span class="wp-block-navigation-item__label">Hosting</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpressfoundation.org/donate/"><span class="wp-block-navigation-item__label">Donate</span></a></li></ul></nav>
+	<div class="wp-container-61e8f8f476fcc wp-block-group global-footer__navigation-container">
+		<nav class="wp-container-61e8f8f4575b7 global-footer__navigation-important wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://de.wordpress.org/about/"><span class="wp-block-navigation-item__label">About</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://de.wordpress.org/news/"><span class="wp-block-navigation-item__label">Blog</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/hosting/"><span class="wp-block-navigation-item__label">Hosting</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpressfoundation.org/donate/"><span class="wp-block-navigation-item__label">Donate</span></a></li></ul></nav>

-		<nav class="wp-container-61e8f8f262225 global-footer__navigation-information wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/support/"><span class="wp-block-navigation-item__label">Support</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://developer.wordpress.org/"><span class="wp-block-navigation-item__label">Developers</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://make.wordpress.org/"><span class="wp-block-navigation-item__label">Get Involved</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://learn.wordpress.org/"><span class="wp-block-navigation-item__label">Learn</span></a></li></ul></nav>
+		<nav class="wp-container-61e8f8f45e246 global-footer__navigation-information wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://de.wordpress.org/support/"><span class="wp-block-navigation-item__label">Support</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://developer.wordpress.org/"><span class="wp-block-navigation-item__label">Developers</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://make.wordpress.org/"><span class="wp-block-navigation-item__label">Get Involved</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://learn.wordpress.org/?locale=de_DE"><span class="wp-block-navigation-item__label">Learn</span></a></li></ul></nav>

-		<nav class="wp-container-61e8f8f2664e8 global-footer__navigation-resources wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/showcase/"><span class="wp-block-navigation-item__label">Showcase</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/plugins/"><span class="wp-block-navigation-item__label">Plugins</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/themes/"><span class="wp-block-navigation-item__label">Themes</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/patterns/"><span class="wp-block-navigation-item__label">Patterns</span></a></li></ul></nav>
+		<nav class="wp-container-61e8f8f464c47 global-footer__navigation-resources wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/showcase/"><span class="wp-block-navigation-item__label">Showcase</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://de.wordpress.org/plugins/"><span class="wp-block-navigation-item__label">Plugins</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://de.wordpress.org/themes/"><span class="wp-block-navigation-item__label">Themes</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://de.wordpress.org/patterns/"><span class="wp-block-navigation-item__label">Patterns</span></a></li></ul></nav>

-		<nav class="wp-container-61e8f8f26a807 global-footer__navigation-community wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://central.wordcamp.org/"><span class="wp-block-navigation-item__label">WordCamp</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.tv/"><span class="wp-block-navigation-item__label">WordPress.TV</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://buddypress.org/"><span class="wp-block-navigation-item__label">BuddyPress</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://bbpress.org/"><span class="wp-block-navigation-item__label">bbPress</span></a></li></ul></nav>
+		<nav class="wp-container-61e8f8f46b5dd global-footer__navigation-community wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://central.wordcamp.org/"><span class="wp-block-navigation-item__label">WordCamp</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.tv/"><span class="wp-block-navigation-item__label">WordPress.TV</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://buddypress.org/"><span class="wp-block-navigation-item__label">BuddyPress</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://bbpress.org/"><span class="wp-block-navigation-item__label">bbPress</span></a></li></ul></nav>

-		<nav class="wp-container-61e8f8f26eafb global-footer__navigation-external wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.com/?ref=wporg-footer"><span class="wp-block-navigation-item__label">WordPress.com</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://ma.tt/"><span class="wp-block-navigation-item__label">Matt</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.org/about/privacy/"><span class="wp-block-navigation-item__label">Privacy</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://publiccode.eu/"><span class="wp-block-navigation-item__label">Public Code</span></a></li></ul></nav>
+		<nav class="wp-container-61e8f8f471f6b global-footer__navigation-external wp-block-navigation"><ul class="wp-block-navigation__container"><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://wordpress.com/?ref=wporg-footer"><span class="wp-block-navigation-item__label">WordPress.com</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://ma.tt/"><span class="wp-block-navigation-item__label">Matt</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://de.wordpress.org/about/privacy/"><span class="wp-block-navigation-item__label">Privacy</span></a></li><li class=" wp-block-navigation-item wp-block-navigation-link"><a class="wp-block-navigation-item__content"  href="https://publiccode.eu/"><span class="wp-block-navigation-item__label">Public Code</span></a></li></ul></nav>
 	</div>
@@ -21,3 +21,3 @@
 		<figure class="wp-block-image is-resized global-footer__wporg-logo-mark">
-			<a href="https://wordpress.org/">
+			<a href="https://de.wordpress.org/">
 				<img src="https://wordpress.org/style/images/w-mark.svg" alt="WordPress.org" width="27" height="27" />
@@ -29,4 +29,4 @@
 		<figure class="wp-block-image is-resized global-footer__wporg-logo-full">
-			<a href="https://wordpress.org/">
-				<img src="https://wordpress.org/wp-content/mu-plugins/pub-sync/blocks/global-header-footer/images/wporg-logo.svg" alt="WordPress.org" width="160" height="24" />
+			<a href="https://de.wordpress.org/">
+				<img src="https://de.wordpress.org/wp-content/mu-plugins/pub-sync/blocks/global-header-footer/images/wporg-logo.svg" alt="WordPress.org" width="160" height="24" />
 			</a>
```

(edit: Oh look, Guthubug. it's not highlighting the + lines in the middle there..)

File: mu-plugins/blocks/global-header-footer/footer.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	<div class="wp-block-group global-footer__navigation-container">
 		<!-- wp:navigation {"orientation":"vertical","className":"global-footer__navigation-important","overlayMenu":"never"} -->
 			<!-- wp:navigation-link {"label":"<?php echo esc_html_x( 'About', 'Menu item title', 'wporg' ); ?>","url":"https://wordpress.org/about/","kind":"custom","isTopLevelLink":true} /-->
-			<!-- wp:navigation-link {"label":"<?php echo esc_html_x( 'Blog', 'Menu item title', 'wporg' ); ?>","url":"https://wordpress.org/news","kind":"custom","isTopLevelLink":true} /-->
+			<!-- wp:navigation-link {"label":"<?php echo esc_html_x( 'Blog', 'Menu item title', 'wporg' ); ?>","url":"https://wordpress.org/news/","kind":"custom","isTopLevelLink":true} /-->
```
Comment (dd32) [inline mu-plugins/blocks/global-header-footer/footer.php:22]: I note the Blog is called Blog in the footer, but News in the header..

The old Header had 'Blog' and it's been replaced with 'News'. 

Rosetta sites currently have 'Blog', but maybe that should also change to News.

**Review by ryelle — APPROVED** [review]
Works as expected 👍🏻

**Comment by angelasjin** [comment]
Let's go with "News".

**Pull request merged**